pkg/kubernetes/util_test.go

```diff
@@ -45,3 +46,49 @@ func Test_ObjectContains(t *testing.T) {
 		assert.Equal(t, testcase.expected, ObjectContains(testcase.object, testcase.value))
 	}
 }
+
+type UnstructuredListContainsTestCase struct {
+	objectlist *unstructured.UnstructuredList
+	value      string
+	expected   bool
+}
+
+func Test_UnstructuredListContains(t *testing.T) {
```

Comment:
This is almost an identical test like `Test_ObjectContains` above. I suggest to remove the `Test_ObjectContains` and keep your version only, otherwise we test the same thing twice, which only confuses future contributors.

Reply:
Indeed, I've removed the `Test_ObjectContains` and added some values in the `metadata` of mock objects to test all switch case in `ObjectContains`.